src/components/Releases/ReleaseOverviewTab.tsx

```diff
@@ -99,7 +116,7 @@ const ReleaseOverviewTab: React.FC = () => {
             <DescriptionListGroup>
               <DescriptionListTerm>Release Process</DescriptionListTerm>
               <DescriptionListDescription>
-                {release.status?.automated ? 'Automatic' : 'Manual'}
```

Comment:
I'd put back the optional chain here, since the `release.status` could be `undefined`